data/recommendations.html

```diff
@@ -50,25 +50,29 @@ <h2>&zwnj;Backups&zwnj;</h2>
                         </div>
                         <div class="text-md-right">
                             <div id="backup-remove">
-                            <a class="btn btn-inverse" onclick="cmd('backup');">
+                            <a class="btn btn-inverse" onclick="cmd('deja-dup');">
                                 &nbsp;<img class="button-icon" src="img/applications/deja-dup.png" />&nbsp;<span
                                 class="button-text">&zwnj;Configure Backup&zwnj;</span>&nbsp;
                             </a>
+                            <a class="btn btn-danger" onclick="cmd('remove?backup');">
+                                <span class="material-icons">&#xE872;</span><span
+                                    class="button-text">&zwnj;Remove&zwnj;</span>
+                            </a>
                             <span id="backup-hint" class="center"><small>
                                 <p><i class="material-icons">&#xE88F;</i> &zwnj;<b>Backups</b> can be found later in the menu
                                     <b>Utilities <i class="material-icons">&#xE315;</i> Backups</b>.&zwnj;
                                 </p>
                                 </small></span>
                             </div>
                             <a id="backup-install" class="btn btn-success" onclick="cmd('install?backup');">&nbsp;<span
-                                class="material-icons">&#xE2C0;</span><span class="button-text">&zwnj;Install&zwnj;</span>&nbsp;</a>
+                                class="material-icons">&#xE2C0;</span><span class="button-text">&zwnj;Install Deja-Dup&zwnj;</span>&nbsp;</a>
```

Comment:
deja-dup has a remove button now ... but others just a install button - need to be consistent all through this page - everything with a remove ... or all just showing "installed" only.